contrib/build_sdk.sh

```diff
@@ -550,13 +636,20 @@ curl_pkg() {
             --disable-shared --with-zlib=$install_dir --enable-ares=$install_dir $openssl_flags"
     fi
 
+    if [ $incremental -eq 1 ] && [ -e $status_dir/$name.success ]; then
+        echo "$name already built"
+        return
+    else
+        rm -f $status_dir/$name.success
+    fi
+
     package_download $name $curl_url $curl_file $curl_md5
     if [ $download_only -eq 1 ]; then
         return
     fi
 
     package_extract $name $curl_file $curl_dir
-    package_configure $name $curl_dir $install_dir "$curl_params"
+    package_configure $name $curl_dir $install_dir "$curl_params" "-ldl -lpthread"
```

Comment:
I'm not 100% sure this has no consequences. does curl always need libpthread? how could it be working?

Reply:
After testing some compilations: As a matter of fact, this breaks some cross compilation for Android in a system where libpthread is not available. If this is required for QNAP/SYNOLOGY compilation should be configured as parameter

Reply:
I ran into a problem linking curl at some point after removing "shared" from the OpenSSL static configuration, and found this solution (probably on stackoverflow).  However I'm not able to recreate the problem now, so perhaps this was addressed by other config fixes later on.  I'll remove these extra libs for this submission.